subiquity/common/apidef.py

```diff
@@ -44,11 +44,12 @@
 @api
 class API:
     """The API offered by the subiquity installer process."""
+    identity = simple_endpoint(IdentityData)
     locale = simple_endpoint(str)
-    proxy = simple_endpoint(str)
     mirror = simple_endpoint(str)
-    identity = simple_endpoint(IdentityData)
+    proxy = simple_endpoint(str)
     ssh = simple_endpoint(SSHData)
+    updates = simple_endpoint(str)
```

Comment:
I'd ever so slightly prefer to have this API method take an enum rather than a string.

Reply:
I messed with putting enum on the API boundary and it made a mess of things.  I'm willing to try again but at a certain point I cut my losses and pushed a simpler version.